src/Requests.jl

```diff
@@ -122,7 +123,7 @@ function mimetype(r::Response)
         ct = split(headers(r)["Content-Type"], ";")[1]
         return Nullable(ct)
     else
-        return Nullable{UTF8String}()
+        return Nullable{String}()
```

Comment:
`Compat.UTF8String` (`String` surprisingly aliases to `ByteString` on 0.3/0.4, not `UTF8String`)